horizons/util/worldobject.py

```diff
@@ -82,4 +82,6 @@ def __lt__(self, other):
 
 	# for testing:
 	@classmethod
-	def get_objs(self): return self.__objects
+	def get_objs(cls):
+		"""returns objects"""
```

Comment:
This comment is not necessary, as it does not add any explanation, but just repeats the method name.